server/src/routes/twiml.ts

```diff
@@ -16,15 +16,29 @@ export function createTwimlRoute(
     }
 
     const { to } = req.body;
-
     if (typeof to !== 'string') {
-      res.status(401).send('Missing "to".');
+      res.status(400).send('Missing "to".');
       return;
     }
 
-    const twimlResponse = new VoiceResponse();
+    const recipientType = (['client', 'number'] as const).find(
+      (r) => r === req.body.recipientType,
+    );
+    if (typeof recipientType === 'undefined') {
+      res.status(400).send('Recipient type invalid.');
+      return;
+    }
+
+    const callerId = recipientType === 'number'
+      ? twilioCredentials.CALLER_ID
+      : req.body.From;
 
-    twimlResponse.dial().client(to);
+    const twimlResponse = new VoiceResponse();
+    const dial = twimlResponse.dial({
+      answerOnBridge: true,
```

Comment:
Why are we setting the default for `answerOnBridge` to true?

Reply:
It seemed appropriate so we could get `ringing` events properly, any reason we shouldn't have it on by default?

Reply:
I don't know. But there must be a reason why they made this 'false' by default. Is it best practice to leave this on? Any side effects?

Reply:
Let's ask the mobile experts.

Reply:
Try asking DevEd as well. What are they usually advising customers?

Reply:
Conclusion is to set it to true. See thread https://twilio.slack.com/archives/C03QLE085NG/p1675281990315379